clusters/build-clusters/02_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml

```diff
@@ -27,12 +27,12 @@ spec:
         - name: AWS_ACCESS_KEY_ID
           valueFrom:
             secretKeyRef:
-              name: aws-ci-infra-ci-build02-audit-logs-uploader-credentials
+              name: aws-build02-audit-logs-uploader-credentials
```

Comment:
There is an about 0% chance this will be used in a step.

Reply:
Whitelist those?
How validation can tell which one are in a step? Reading ci-op configs?

Reply:
Which validation? This is a standalone daemonset, it shouldn't have any validation. It is even in an entirely different namespace

Reply:
I should just do it for test-credentials.